ngx_http_dav_ext_module.c

```diff
@@ -328,16 +328,9 @@ ngx_http_dav_ext_send_propfind_atts(ngx_http_request_t *r,
 						"<D:displayname>"
 			);
 
-		if (uri->len) {
-
-			for(name.data = uri->data + uri->len;
-				name.data >= uri->data + 1 && name.data[-1] != '/'; 
-				--name.data);
-
-			name.len = uri->data + uri->len - name.data;
-
-			NGX_HTTP_DAV_EXT_OUTES(&name);
-		}
+		name.data = strrchr(path, '/') + 1;
+		name.len = strlen(name.data);
+		NGX_HTTP_DAV_EXT_OUTES(&name);
```

Comment:
this will use `NGX_HTTP_DAV_EXT_ESCAPE` which in turn will end up using `ngx_escape_html` so it should be ok to pass in encoding used by filesystem which i assume here is UTF8 (to be really correct i guess one should check locale for encoding). I think and hope `ngx_escape_html` encodes correct to be safe for XML, for HTML is another story.